core/pom.xml

```diff
@@ -193,4 +193,56 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>aws</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.iceberg</groupId>
```

Comment:
It would be better to add these dependencies to parent <dependencyManagement>

Reply:
If moving to parent, it will be depended by all modules.